packages/cli/src/runners/errors/task-runner-restart-loop-error.ts

```diff
@@ -0,0 +1,14 @@
+import { ApplicationError } from 'n8n-workflow';
+
+export class TaskRunnerRestartLoopError extends ApplicationError {
+	constructor(
+		public readonly howManyTimes: number,
+		public readonly timePeriodMs: number,
+	) {
+		const message = `Task runner has restarted ${howManyTimes} times within ${timePeriodMs / 1000} seconds`;
```

Comment:
Let's add actionable advice for user?

Reply:
Any suggestion what that could be?

Reply:
`Task runner has restarted n times within y seconds. This is an abnormally high restart rate that suggests a bug is preventing your runner process from starting up. Please file a report at: https://github.com/n8n-io/n8n/issues`